elide-contrib/elide-dynamic-config-helpers/src/test/java/com/yahoo/elide/contrib/dynamicconfighelpers/validator/DynamicConfigValidatorTest.java

```diff
@@ -107,7 +107,7 @@ public void testBadSecurityRoleConfig() {
 
     @Test
     public void testBadTableConfigJoinType() {
-        assertThrows(ValueInstantiationException.class, () -> DynamicConfigValidator
+        assertThrows(NullPointerException.class, () -> DynamicConfigValidator
```

Comment:
is this change needed?

Reply:
i don't see any other related change in main class.

Reply:
the function throws NPE. hence the change.

Reply:
It looks very odd for a test to _expect_ a NPE.  NPE should never happen in code - so this test looks wrong or something is off.